### DISCUSSION TITLE
[feature-request] Tab auto-complete suggestion/hint popup

### DISCUSSION THREAD

**Discussion by craziedde** [espanso/espanso#1535] [Feature requests and Ideas]
Hello,

I have a feature request:
Possible to add "Tab" auto-complete for "trigger" suggestion/hint popup
Or auto-expend to the matching "trigger"

![image](https://user-images.githubusercontent.com/2974843/220737303-dc795b9f-0731-49a3-aa1b-d89fcceabf79.png)

**Comment by smeech** [comment]
Sorry about the late reply - I am trying to mop up and sort the old Issues and Discussions.

There have been several requests to improve the Search facility, and yours is a good one to consider.

**Reply by craziedde** [reply]
Sounds good, let me know if you need me to provide more details on the request
